### PR TITLE
coast.rst: -M argument, mention FORMAT_FLOAT_OUT to reduce number of …

### PR DESCRIPTION
**Pull request by anbj**
Address #8516.

### CODE REVIEW

**Comment by anbj** [comment]
Dont approve! Must change link.

**Comment by anbj** [comment]
Ok, good to go.

File: doc/rst/source/coast.rst [hunk 1]
```diff
@@ -219,6 +219,10 @@ Optional Arguments
     just certain levels (1-4) then use the full syntax **-W**\ *level*/\ *pen*
     and repeat for each level (pen is not used but required to parse the level correctly).
 
+    Since the resolution of the datasets typically are in the order
+    of 100s of meter, consider reducing the number of decimals by
+    setting :term:`FORMAT_FLOAT_OUT`.
```
Comment (Esteban82) [inline doc/rst/source/coast.rst:224]: I found "100s" to be a bit confusing. I first tought in 100 sec-arc. I would prefer "hundreds of meters"

Comment (anbj) [reply doc/rst/source/coast.rst:224]: Agree - changing.

Comment (anbj) [reply doc/rst/source/coast.rst:224]: Please check now.

**Review by Esteban82 — APPROVED** [review]

**Pull request merged**